### ISSUE TITLE
[`BUG`] : `.py` files inside `docs` folder is included in `wheel`

### ISSUE THREAD

**Issue by baseplate-admin** [requests-cache/aiohttp-client-cache#133]
Thanks for making this awesome library,
---
 
### The problem
I am not sure if you are aware of this but the `.py` file in `docs` folder is getting bundled into python wheels.

### Expected behavior
We should ignore `.py` file inside `docs` folder

### Steps to reproduce the behavior
1. `pip install aiohttp-client-cache`

<details> 
<summary> A Picture to demonstrate the issue </summary>

![image](https://user-images.githubusercontent.com/61817579/191805570-c5039b31-da2d-4a7e-ba3a-5cd5c830369c.png)

</details>

### Workarounds
Nope

### Environment
- aiohttp-client-cache version: 0.7.3
- Python version: 3.10
- Platform: Windows 11

---

Thanks for reading. I wish you a good day

**Comment by JWCook** [comment]
This is actually intentional; the included folder is [aiohttp_client_cache/docs](https://github.com/requests-cache/aiohttp-client-cache/tree/main/aiohttp_client_cache/docs), which is used at runtime for adding accurate signatures and type hints for some backend-specific connection objects. For example, if you want to pass [Redis connection parameters](https://redis-py.readthedocs.io/en/stable/connections.html) to the Redis backend, you'll get autocompletion and type checking for `username`, `password`, etc.:
```python
backend = RedisBackend(username='xx', password='xx')
```

That's separate from the root [docs](https://github.com/requests-cache/aiohttp-client-cache/tree/main/docs) folder, which is only used for Sphinx documentation builds (hosted at https://aiohttp-client-cache.readthedocs.io).

Does that make sense?

**Comment by baseplate-admin** [comment]
Yep it does. Thanks for quick reply :D 

Sorry for the bad ping. I wish you a good day :}

**Comment by JWCook** [comment]
> Sorry for the bad ping. 

Not at all, more eyes on the code is always helpful!

Can you think of a better name for that folder?

**Comment by baseplate-admin** [comment]
Hi there,

First of all thanks again for this project. It removed a ton of headache while providing a good customizable api. I used both of your project and both of them are awesome.

---
Now back to business.

I took another look at the files.

i have a few observations:
1. I have never seen `docs` folder included in wheel build. ( PyDantic does auto completion too but somehow its backed into core module ? )
2. Maybe we can make a file named `autocomplete.py` in `aiohttp_client_cache` ?

I am quite new to programming. So no worries if my opinion doesn't make much sense.

Thanks again. I wish you a good day
